modules/nw-infw-operator-config-object.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: CONCEPT
+[id="nw-infw-operator-config-object_{context}"]
+== Ingress Node Firewall configuration object
+
+The fields for the Ingress Node Firewall configuration object are described in the following table:
+
+.Ingress Node Firewall Configuration object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the CR object. The name of the firewall rules object must be `ingressnodefirewallconfig`.
+
+|`metadata.namespace`
+|`string`
+|Namespace for the Ingress Firewall Operator CR object. The `IngressNodeFirewallConfig` CR must be created inside the `openshift-ingress-node-firewall` namespace.
+
+|`spec.nodeSelector`
+|`string`
+|
+A node selection constraint used to target nodes through specified node labels. For example:
+
+[source,yaml]
+----
+spec:
+  nodeSelector:
+    node-role.kubernetes.io/worker: ""
+----
+
+[NOTE]
+====
+One label used in `nodeSelector` must match a label on the nodes in order for the daemon set to start. For example, if the node labels `node-role.kubernetes.io/worker` and `node-type.kubernetes.io/vm` are applied to a node, then at least one label must be set using `nodeSelector` for the daemon set to start.
+====
+
+|====
+
+[NOTE]
+====
+The Operator consumes the CR and creates an ingress node firewall daemon set on all the nodes that match the `nodeSelector`.
+====
+
+[discrete]
+[id="nw-ingress-node-firewall-example-cr-2_{context}"]
+== Ingress Node Firewall Operator example configuration
```

Comment:
@JoeAldinger Could the title be "Ingress Node Firewall Configuration example" ? It is not operator 's configuration.

Reply:
I will fix.

Reply:
Thanks!